internal/stdio/dial.go

```diff
@@ -0,0 +1,126 @@
+package dialstdio
+
+import (
+	"io"
+	"net"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+func Action() error {
+	// Fixme ? Socket address is hard coded
+	// https://github.com/moby/buildkit/blob/c172172ea17c7df80955f3f18aed7473fb0ade33/examples/buildctl-daemonless/buildctl-daemonless.sh
+	addr := "unix:///run/buildkit/buildkitd.sock"
```

Comment:
this is okay since we are running in a container and can always use that path, but lets make it a const